repobee_sanitizer/sanitizer.py

```diff
@@ -77,9 +84,24 @@ def command(self) -> Optional[plug.Result]:
             errors = _sanitize_repo.sanitize_files(repo_root, file_relpaths)
         else:
             LOGGER.info(f"Sanitizing repo and updating {self.target_branch}")
+
+            try:
+                effective_target_branch = (
+                    self._resolve_effective_target_branch(repo_root)
+                )
+            except _sanitize_repo.EmptyTargetBranchError:
+                return plug.Result(
+                    name="sanitize-repo",
+                    msg=f"Can not create a pull request branch from empty "
```

Comment:
Possibly a good idea to change the msg to "empty or non-existing"

Reply:
There's no such thing as an "empty branch". A branch is just a non-empty file in `.git/refs/heads`, and so an "empty branch" would be an empty file in there, which is and invalid state for Git. A branch is only a branch if it has at least one commit. Mind you it can be empty of _content_, but it can't be empty of commits.

So, this message should read "non-existing" only.